src/xrpld/rpc/handlers/AccountObjects.cpp

```diff
@@ -220,6 +220,9 @@ doAccountObjects(RPC::JsonContext& context)
     {
         auto [rpcStatus, type] = RPC::chooseLedgerEntryType(params);
 
+        if (!RPC::isAccountObjectsValidType(type))
```

Comment:
Looks like this isAccountObjectsValidType function is used in this handler only. We can just add a function/lambda here ?